documents-webapp/src/main/webapp/vue-app/documents/components/DocumentsMain.vue

```diff
@@ -216,7 +216,9 @@ export default {
       .finally(() => {
         this.checkDefaultViewOptions();
         this.optionsLoaded = true;
-        this.refreshFiles()
+        const queryParams = new URLSearchParams(window.location.search);
+        const disablePreview = queryParams.has('path');
+        this.refreshFiles(null, null, null, null, null, disablePreview)
```

Comment:
Could you refactor the function refreshFiles to take an object as parameter intead of the current 6 parameters ?